### PR TITLE
Tests for OneAnd

### PR DESCRIPTION
**Pull request by adelbertc**

### CODE REVIEW

**Comment by non** [comment]
Looks great to me. :+1:

File: core/src/main/scala/cats/data/OneAnd.scala [hunk 1]
```diff
@@ -85,7 +85,7 @@ trait OneAndInstances {
   implicit def oneAndShow[A, F[_]](implicit A: Show[A], FA: Show[F[A]]): Show[OneAnd[A, F]] =
     Show.show[OneAnd[A, F]](_.show)
 
-  implicit def oneAndFunctor[F[_]](F: Functor[F]): Functor[OneAnd[?, F]] =
+  implicit def oneAndFunctor[F[_] : Functor](implicit F: Functor[F]): Functor[OneAnd[?, F]] =
```
Comment (ceedubs) [inline core/src/main/scala/cats/data/OneAnd.scala]: Looks like the `Functor` requirement is stated twice.

**Comment by ceedubs** [comment]
Looks good to me after the minor change I pointed out for `oneAndFunctor`.

**Comment by ceedubs** [comment]
Thanks!

**Pull request merged**